bundle/compliance/cis_aws/rules/cis_1_17/test.rego

```diff
@@ -0,0 +1,51 @@
+package compliance.cis_aws.rules.cis_1_17
+
+import data.compliance.cis_aws.data_adapter
+import data.lib.test
+
+generate_input(roles) = {
+	"subType": "aws-policy",
```

Comment:
I wonder if the focus should be on the role, in our findings the user would like to see which role is holding the policy.
In that case the `sub-type` should be `aws-role` and the id should be the role ARN.
wdyt?

Reply:
I don't think that's possible, the rule fails when there are no roles that attach this policy. If there is no rule, we can't use it as the resource.